src/theme/components/header/headerLogin.ts

```diff
@@ -10,7 +10,7 @@ const loginFocus = () => ({
 const HeaderLogin = {
   parts: ["logoutButton", "patronGreeting"],
   baseStyle: ({ patronName }) => ({
-    bg: { base: "ui.black", md: headerBlue },
+    bg: { base: headerBlack, md: headerBlue },
     boxShadow: "2px 2px 3px 4px rgb(100 100 100 / 25%)",
     display: { base: "block", md: "flex" },
     flexDirection: "column",
```

Comment:
There is a `marginTop: { md: “1px” }` that is added on line 18.  Why was that added?  With that style in place, I am seeing a 1px gap between the dropdown and the Close button.

![image](https://user-images.githubusercontent.com/5241828/183439431-b2c84728-b4cc-4ee0-bfbf-339bfb4a22c5.png)

Reply:
It's added because that's how it appears in the current NYPL Header.

Reply:
In Chrome, I do not see that same gap on NYPL.org.

![image](https://user-images.githubusercontent.com/5241828/183458895-b67b370e-331b-4d34-a97a-55697c8a89d8.png)